diff --git a/node_modules/@keystone-next/admin-ui/pages/App/dist/admin-ui.cjs.dev.js b/node_modules/@keystone-next/admin-ui/pages/App/dist/admin-ui.cjs.dev.js
index e7d181c..5bd4694 100644
--- a/node_modules/@keystone-next/admin-ui/pages/App/dist/admin-ui.cjs.dev.js
+++ b/node_modules/@keystone-next/admin-ui/pages/App/dist/admin-ui.cjs.dev.js
@@ -34,11 +34,24 @@ function _interopDefault (e) { return e && e.__esModule ? e : { 'default': e };
 
 var React__default = /*#__PURE__*/_interopDefault(React);
 
+const RenderThemeProvider = ({
+  children,
+  theme
+}) => {
+  return theme && /*#__PURE__*/React__default['default'].createElement(core.ThemeProvider, {
+    theme: theme
+  }, children) || children;
+};
+
 const getApp = props => ({
   Component,
   pageProps
 }) => {
-  return /*#__PURE__*/React__default['default'].createElement(core.Core, null, /*#__PURE__*/React__default['default'].createElement(context_dist_adminUi.KeystoneProvider, props, /*#__PURE__*/React__default['default'].createElement(Errors.ErrorBoundary, null, /*#__PURE__*/React__default['default'].createElement(Component, pageProps))));
+  var _props$adminConfig;
+
+  return /*#__PURE__*/React__default['default'].createElement(core.Core, null, /*#__PURE__*/React__default['default'].createElement(context_dist_adminUi.KeystoneProvider, props, /*#__PURE__*/React__default['default'].createElement(RenderThemeProvider, {
+    theme: (_props$adminConfig = props.adminConfig) === null || _props$adminConfig === void 0 ? void 0 : _props$adminConfig.theme
+  }, /*#__PURE__*/React__default['default'].createElement(Errors.ErrorBoundary, null, /*#__PURE__*/React__default['default'].createElement(Component, pageProps)))));
 };
 
 exports.getApp = getApp;
diff --git a/node_modules/@keystone-next/admin-ui/pages/App/dist/admin-ui.cjs.prod.js b/node_modules/@keystone-next/admin-ui/pages/App/dist/admin-ui.cjs.prod.js
index d4d96a6..658e5cc 100644
--- a/node_modules/@keystone-next/admin-ui/pages/App/dist/admin-ui.cjs.prod.js
+++ b/node_modules/@keystone-next/admin-ui/pages/App/dist/admin-ui.cjs.prod.js
@@ -34,11 +34,24 @@ function _interopDefault (e) { return e && e.__esModule ? e : { 'default': e };
 
 var React__default = /*#__PURE__*/_interopDefault(React);
 
+const RenderThemeProvider = ({
+  children,
+  theme
+}) => {
+  return theme && /*#__PURE__*/React__default['default'].createElement(core.ThemeProvider, {
+    theme: theme
+  }, children) || children;
+};
+
 const getApp = props => ({
   Component,
   pageProps
 }) => {
-  return /*#__PURE__*/React__default['default'].createElement(core.Core, null, /*#__PURE__*/React__default['default'].createElement(context_dist_adminUi.KeystoneProvider, props, /*#__PURE__*/React__default['default'].createElement(Errors.ErrorBoundary, null, /*#__PURE__*/React__default['default'].createElement(Component, pageProps))));
+  var _props$adminConfig;
+
+  return /*#__PURE__*/React__default['default'].createElement(core.Core, null, /*#__PURE__*/React__default['default'].createElement(context_dist_adminUi.KeystoneProvider, props, /*#__PURE__*/React__default['default'].createElement(RenderThemeProvider, {
+    theme: (_props$adminConfig = props.adminConfig) === null || _props$adminConfig === void 0 ? void 0 : _props$adminConfig.theme
+  }, /*#__PURE__*/React__default['default'].createElement(Errors.ErrorBoundary, null, /*#__PURE__*/React__default['default'].createElement(Component, pageProps)))));
 };
 
 exports.getApp = getApp;
diff --git a/node_modules/@keystone-next/admin-ui/pages/App/dist/admin-ui.esm.js b/node_modules/@keystone-next/admin-ui/pages/App/dist/admin-ui.esm.js
index 8a5e360..0711a31 100644
--- a/node_modules/@keystone-next/admin-ui/pages/App/dist/admin-ui.esm.js
+++ b/node_modules/@keystone-next/admin-ui/pages/App/dist/admin-ui.esm.js
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Core } from '@keystone-ui/core';
+import { Core, ThemeProvider } from '@keystone-ui/core';
 import '@babel/runtime/helpers/extends';
 import '@babel/runtime/helpers/objectWithoutProperties';
 import 'next/router';
@@ -26,11 +26,24 @@ import '@keystone-next/types';
 import '../../../dist/admin-meta-graphql-87ba5444.esm.js';
 import '@keystone-next/admin-ui-utils';
 
+const RenderThemeProvider = ({
+  children,
+  theme
+}) => {
+  return theme && /*#__PURE__*/React.createElement(ThemeProvider, {
+    theme: theme
+  }, children) || children;
+};
+
 const getApp = props => ({
   Component,
   pageProps
 }) => {
-  return /*#__PURE__*/React.createElement(Core, null, /*#__PURE__*/React.createElement(KeystoneProvider, props, /*#__PURE__*/React.createElement(ErrorBoundary, null, /*#__PURE__*/React.createElement(Component, pageProps))));
+  var _props$adminConfig;
+
+  return /*#__PURE__*/React.createElement(Core, null, /*#__PURE__*/React.createElement(KeystoneProvider, props, /*#__PURE__*/React.createElement(RenderThemeProvider, {
+    theme: (_props$adminConfig = props.adminConfig) === null || _props$adminConfig === void 0 ? void 0 : _props$adminConfig.theme
+  }, /*#__PURE__*/React.createElement(ErrorBoundary, null, /*#__PURE__*/React.createElement(Component, pageProps)))));
 };
 
 export { getApp };
diff --git a/node_modules/@keystone-next/admin-ui/pages/ListPage/dist/admin-ui.cjs.dev.js b/node_modules/@keystone-next/admin-ui/pages/ListPage/dist/admin-ui.cjs.dev.js
index 9207164..ed8fdb1 100644
--- a/node_modules/@keystone-next/admin-ui/pages/ListPage/dist/admin-ui.cjs.dev.js
+++ b/node_modules/@keystone-next/admin-ui/pages/ListPage/dist/admin-ui.cjs.dev.js
@@ -778,7 +778,7 @@ function SortSelectionPopoverContent({
   }));
 }
 
-function useFilters(list) {
+exports.useFilters = function useFilters(list) {
   const {
     query
   } = router.useRouter();
diff --git a/node_modules/@keystone-next/admin-ui/pages/ListPage/dist/admin-ui.cjs.prod.js b/node_modules/@keystone-next/admin-ui/pages/ListPage/dist/admin-ui.cjs.prod.js
index 60dee2b..85e8f8a 100644
--- a/node_modules/@keystone-next/admin-ui/pages/ListPage/dist/admin-ui.cjs.prod.js
+++ b/node_modules/@keystone-next/admin-ui/pages/ListPage/dist/admin-ui.cjs.prod.js
@@ -778,7 +778,7 @@ function SortSelectionPopoverContent({
   }));
 }
 
-function useFilters(list) {
+exports.useFilters =  function useFilters(list) {
   const {
     query
   } = router.useRouter();
diff --git a/node_modules/@keystone-next/admin-ui/pages/ListPage/dist/admin-ui.esm.js b/node_modules/@keystone-next/admin-ui/pages/ListPage/dist/admin-ui.esm.js
index 661e92c..d2553ff 100644
--- a/node_modules/@keystone-next/admin-ui/pages/ListPage/dist/admin-ui.esm.js
+++ b/node_modules/@keystone-next/admin-ui/pages/ListPage/dist/admin-ui.esm.js
@@ -1440,4 +1440,4 @@ const TableBodyCell = props => {
   }, props));
 };
 
-export { getListPage };
+export { getListPage, useFilters };
